redhat-developer/developers.redhat.com
# DEVELOPER-4018 - Research why the code for the Thank you message is not firing for media downloads

https://issues.jboss.org/browse/DEVELOPER-4018

To test: 
* Visit http://rhdp-jenkins-slave.lab4.eng.bos.redhat.com:37082/promotions/building-reactive-microservices-in-java/ and download the book.
* You will be redirected back to developers.redhat.com. 
* Change the domain to http://rhdp-jenkins-slave.lab4.eng.bos.redhat.com:37082/, but keep the media-download-confirmation and url query string intact.
* The download should automatically fire now.
